src/core/include/openvino/op/util/variable_value.hpp

```diff
@@ -32,20 +31,14 @@ class OPENVINO_API VariableValue {
     explicit VariableValue(const ov::Tensor& value);
 
     /// \brief Constructor for VariableValue.
-    /// \deprecated This method is deprecated and will be removed in 2024.0 release. Please use method with ov::Tensor
-    /// instead
```

Comment:
Looks like this API should be removed

@ilya-lavrenov

Reply:
Looks like the current change is valid, because the PR https://github.com/openvinotoolkit/openvino/commit/b2a2266f603c3372a2c2f69341d96bce99c8dd96 introduces deprecation of methods with `ngraph::HostTensorPtr`

Reply:
IMO it was _copy & paste_ slip when adding `ov::Tensor` support into this class.